crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -132,6 +156,7 @@ pub enum TypedTransactionRequest {
     Legacy(TxLegacy),
     EIP2930(TxEip2930),
     EIP1559(TxEip1559),
+    EIP4844(TxEip4844),
```

Comment:
This is what we decode in send_rawTransaction right?
This needs to include the sidecar eventually.

for this

https://github.com/foundry-rs/foundry/blob/master/crates/anvil/src/eth/api.rs#L942-L942
we need another helper type that supports blobs rlp format